oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyState.java

```diff
@@ -47,64 +38,24 @@
 import org.apache.jackrabbit.oak.plugins.memory.StringPropertyState;
 import org.apache.jackrabbit.oak.plugins.value.Conversions;
 import org.jetbrains.annotations.NotNull;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * PropertyState implementation with lazy parsing of the JSOP encoded value.
  */
 final class DocumentPropertyState implements PropertyState {
 
-    private static final Logger LOG = LoggerFactory.getLogger(DocumentPropertyState.class);
-
     private final DocumentNodeStore store;
 
     private final String name;
 
     private final String value;
 
     private PropertyState parsed;
-    private final byte[] compressedValue;
-    private final Compression compression;
-
-    private static int COMPRESSION_THRESHOLD = SystemPropertySupplier
-            .create("oak.documentMK.stringCompressionThreshold ", -1).loggingTo(LOG).get();
 
     DocumentPropertyState(DocumentNodeStore store, String name, String value) {
-        this(store, name, value, Compression.GZIP);
-    }
-
-    DocumentPropertyState(DocumentNodeStore store, String name, String value, Compression compression) {
```

Comment:
Presumably this class should be reverted to the state before compression? In doing that it looks like it missed a commit that came in between : b04de7ea9dce0859de09f90964fa650a7bf5a978

Reply:
Added that commit